requirements.txt

```diff
@@ -1,55 +1,108 @@
+aiosqlite==0.18.0
```

Comment:
its best to only list the packages you pip installed, and not include those packages' dependencies. If we want to pin the the minor version of prefect, that'd be

```
prefect==2.8.*
```